### PR TITLE
Update main.js

### PR DESCRIPTION
**Pull request by reiniergs**
It's a litter change of forEach to every, There is no way to stop or break a forEach() loop. it makes no sense to continue when we already find an invalid component.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -365,17 +365,18 @@ Formsy.Form = React.createClass({
...
     // We need a callback as we are validating all inputs again. This will
     // run when the last component has set its state
     var onValidationComplete = function () {
-      inputKeys.forEach(function (name) {
+      allIsValid = inputKeys.every(function (name) {
         if (!inputs[name].state._isValid) {
-          allIsValid = false;
+          return  false;
         }
+        return true;
```
Comment (Semigradsky) [inline src/main.js]: This code can be a bit easier:
```js
var allIsValid = inputKeys.every(function (name) {
         return !inputs[name].state._isValid;
})
```

Comment (reiniergs) [inline src/main.js]: Yes, that's correct.

Comment (Semigradsky) [inline src/main.js]: Can you change? I will merge PR after this.photos

Comment (reiniergs) [inline src/main.js]: Done :)

**Comment by Semigradsky** [comment]
Nice! Thanks for assistance!

**Pull request merged**